src/config/dynamic-theme-fixes.config

```diff
@@ -8136,6 +8136,9 @@ web.whatsapp.com
 INVERT
 .selectable-text > span._3PxOr
 span[data-icon="audio-download"]
+span[dir="ltr"] > ._2ezGC
+div._1PTz1 > span._2ezGC
```

Comment:
This inverts the icon inside the QR-Code?

I don't see why, due to the [Error correction math](Reed–Solomon error correction) in the QR-Code standard it won't matter at all if it's inverted or not it should work correctly.

Reply:
Remove this one Line and add under `IGNORE INLINE STYLE`
```CSS
.landing-main div[data-ref] span > svg > path
```